File: app/src/main/java/ga/forntoh/bableschool/model/misc/TopStudent.java
package ga.forntoh.bableschool.model.misc;

import ga.forntoh.bableschool.utils.Utils;

@SuppressWarnings("unused")
public class TopStudent {

    private String image, name, surname, school;
    private double average;

    public String getImage() {
        return image;
    }

    public String getName() {
        return Utils.capEachWord(name);
    }

    public String getSurname() {
        return surname;
    }

    public String getSchool() {
        return school;
    }

    public double getAverage() {
        return average;
    }
}

File: app/src/main/java/ga/forntoh/bableschool/data/model/other/TopStudent.kt
package ga.forntoh.bableschool.data.model.other

import androidx.room.ColumnInfo
import androidx.room.Entity

@Entity(primaryKeys = ["name", "surname"])
data class TopStudent(
        @ColumnInfo var name: String = "",
        @ColumnInfo var surname: String = "",
        @ColumnInfo var image: String? = "",
        @ColumnInfo var school: String? = "",
        @ColumnInfo var average: Double = 0.toDouble()
)